lnd_methods/offchain/get_pending_payments.js

```diff
@@ -0,0 +1,190 @@
+const asyncAuto = require('async/auto');
+const {returnResult} = require('asyncjs-util');
+
+const {isLnd} = require('./../../lnd_requests');
+const {rpcPaymentAsPayment} = require('./../../lnd_responses');
+const {sortBy} = require('./../../arrays');
+
+const defaultLimit = 250;
+const {isArray} = Array;
+const isPending = payment => !!payment && payment.status !== 'SUCCEEDED' && payment.status !== 'FAILED';
```

Comment:
why wouldn't this be `status === IN_FLIGHT`?

Reply:
is on purpose https://api.lightning.community/#lnrpcpaymentpaymentstatus really not sure what are the cases for UNKNOWN but all pending are payments that has not succeeded or failed

Reply:
I think if you don't put in-flight then you might get some succeeded or failed payments show up in here, either through unknown or through an api change down the road